privacyidea/lib/policydecorators.py

```diff
@@ -568,23 +578,23 @@ def auth_otppin(wrapped_function, *args, **kwds):
             user_object=User("", realm="")
         # get the policy
         policy_object = g.policy_object
-        otppin_list = policy_object.get_action_values(ACTION.OTPPIN,
+        otppin_dict = policy_object.get_action_values(ACTION.OTPPIN,
                                                       scope=SCOPE.AUTH,
                                                       realm=user_object.realm,
                                                       resolver=user_object.resolver,
                                                       user=user_object.login,
                                                       client=clientip,
-                                                      unique=True)
```

Comment:
I was thinking: In some cases, it might improve readability to have
```python
otppin_dict = policy_object.get_action_values(...)
otppin_list  = list(otppin_dict.keys())
```
and then use ``otppin_list`` in the following. In fact this also improves performance a little bit because we only construct the list once, not three times. What do you think?

Reply:
On my long and winding way during this PR I was also thinking, if the ``get_action_values`` should return some kind of tuple like

   action_values, policy_name_dict = get_action_values()

to directly have the list of the action_values without the need to use the ``keys()``.